whylog/constraints/__init__.py

```diff
@@ -126,12 +130,32 @@ def verify(self, group_contents, param_dict):
 
         pass
 
+    def _convert_params_values(self):
+        problems = []
+        for param_name, converter_type in six.iteritems(self.PARAMS):
+            param_val = self.params.get(param_name)
+            if param_val is not None:
+                converter = get_converter(converter_type)
+                try:
+                    converted_val = converter.convert(param_val)
+                except ValueError:
```

Comment:
In DateConverter you use dateutil.parser which can raise an `AttributeError`, `NameError`, `TypeError`. `Int` and `float` function can raise `TypeError`. Try: `int(None)` or `float(None)`. I guess this case can appear in many situations and we need to handle it properly. Create new ConverterException class and add for every of this situations inheriting class. In this line you should only catch ConverterException. In every converter you should catch TypeError, ValueError and etc. and raise ConverterException. Which can be catch here and adds to problem.